Sources/UnleashProxyClientSwift/UnleashProxyClientSwift.swift

```diff
@@ -40,19 +40,22 @@ public class UnleashClientBase {
     var metrics: Metrics
 
     public init(unleashUrl: String, clientKey: String, refreshInterval: Int = 15, metricsInterval: Int = 30, disableMetrics: Bool = false, appName: String = "unleash-swift-client", environment: String? = nil, poller: Poller? = nil, metrics: Metrics? = nil) {
+        guard let url = URL(string: unleashUrl), url.scheme != nil else {
```

Comment:
make sure URL passed to the client is correct so that we eliminate impossible URLs down the line